incubator/hnc/internal/reconcilers/object.go

```diff
@@ -382,6 +388,24 @@ func (r *ObjectReconciler) syncPropagated(ctx context.Context, log logr.Logger,
 	return actionNop, nil
 }
 
+func (r *ObjectReconciler) getSelector(ctx context.Context, log logr.Logger, inst *unstructured.Unstructured) labels.Selector {
+	annot := inst.GetAnnotations()
+	selectorstr, present := annot[selectorAnnot]
```

Comment:
Prefer camel case (`selectorStr`). Also, if you liked, you could say:

```
selectorStr, ok := inst.GetAnnotations()[selectorAnnot]
```

The `ok` name is often used to see if an entry exists in a map.

Reply:
done